File: fizzbuzz.js
function fizzbuzz(start, end){
  listNum =[];
  for(i = start; i<=end; i++){
    if (i % 3 == 0 && i % 5 ==0){
      listNum.push('FizzBuzz')
    }else if (i % 3 == 0){
      listNum.push('Fizz')
    } else if (i % 5 == 0){
      listNum.push('Buzz')
    } else {
    listNum.push(i)
    }
  }
  console.log(listNum)
}
fizzbuzz(1, 100)

File: fizzbuzz.rb
require 'pry'
def fizzbuzz
  start_num = 1
  end_num = 15

  list_num = (start_num .. end_num).to_a 
  list_num.map do |num| 
    if num % 3 == 0 && num % 5 == 0
      puts 'fizzbuzz'
    elsif num % 5 == 0
      puts 'buzz'
    elsif num % 3 == 0
      puts 'fizz'
    else
      puts num
    end
  end
end

fizzbuzz